### PR TITLE
Adding datadog docs to ext logForwarding

### PR DESCRIPTION
**Pull request by hegderohit89**

### CODE REVIEW

**Review by sleslie23 — DISMISSED** [review]
I made a few minor copy edits. Other than that it looks good.

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -299,6 +302,34 @@ Example output plugin configuration:
...
+.Datadog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | The host name of the remote Datadog server. | Yes | `http-intake.logs.datadoghq.com`
+| Api Key | The Datadog API key. | Yes |
+| Compress | Compresses the payload in GZIP format. Datadog recommends setting this to `gzip`. | No | `gzip`
+| Datadog Service | The name of the service generating the logs. For example, `runtime_fabric` | No |
+| Datadog Source | The type of service. For example, `postgres` or `nginx` | No |
+| Datadog Tags | The tags assigned to the logs in Datadog. | No |
+| TLS | Enables or disables TLS support. | No | Off
```
Comment (sarjeet2013) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: nit: `Enables or disables TLS support.` -> `End-to-end security communications security protocol support.`

**Review by sarjeet2013 — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 1 above)
Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Ya that makes more sense, specially for datadog case

**Review by sarjeet2013 — DISMISSED** [review]

**Review by hannanelson — DISMISSED** [review]

**Review by sarjeet2013 — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -299,6 +302,34 @@ Example output plugin configuration:
...
+.Datadog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | The host name of the remote Datadog server. | Yes |
+| Api Key | The Datadog API key. | Yes |
+| Compress | Compresses the payload in GZIP format. Datadog recommends setting this to `gzip`. | No | `gzip`
+| Datadog Service | The name of the service generating the logs. For example, `runtime_fabric` | No |
+| Datadog Source | The type of service. For example, `postgres` or `nginx` | No |
+| Datadog Tags | The tags assigned to the logs in Datadog. | No |
+| TLS | End-to-end secure communications protocol support. | No | Off
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]:
```suggestion
| TLS | End-to-end secure communications protocol support.  DataDog does not support CA certificate files.| No | Off
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: @hegderohit89  added as part of table instead of separate note.

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 3]
```diff
@@ -299,6 +302,34 @@ Example output plugin configuration:
...
+.Datadog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | The host name of the remote Datadog server. | Yes |
+| Api Key | The Datadog API key. | Yes |
+| Compress | Compresses the payload in GZIP format. Datadog recommends setting this to `gzip`. | No | `gzip`
+| Datadog Service | The name of the service generating the logs. For example, `runtime_fabric` | No |
+| Datadog Source | The type of service. For example, `postgres` or `nginx` | No |
+| Datadog Tags | The tags assigned to the logs in Datadog. | No |
+| TLS | End-to-end secure communications protocol support.  DataDog does not support CA certificate files.| No | Off
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]:
```suggestion
| TLS | End-to-end secure communications protocol support. DataDog does not support CA certificate files.| No | Off
```

**Review by IsaacEldridge — DISMISSED** [review]

**Review by kmendiratta-crm — APPROVED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**